test/integration_test.js

```diff
@@ -155,6 +156,30 @@ describe("Integration tests", function() {
       });
     });
 
+    describe("#deleteBuckets()", () => {
+      before(function() {
+        try {
+          checkVersion(server.http_api_version, "1.4", "2.0");
+        } catch(err) {
+          this.skip();
```

Comment:
No, this is a test ran contextually depending on the http api version used by the server, as we decided we wanted to go that route.

Basically, before running the test, it checks for the protocol version used by the server, and depending on it will run or skip running the test.